nikola/nikola.py

```diff
@@ -836,6 +836,13 @@ def __init__(self, **config):
             utils.LOGGER.error('CATEGORY_PAGES_FOLLOW_DESTPATH requires CATEGORY_ALLOW_HIERARCHIES = True, CATEGORY_OUTPUT_FLAT_HIERARCHY = False.')
             sys.exit(1)
 
+        # The Utterances comment system has a required configuration value
+        if self.config.get('COMMENT_SYSTEM') == 'utterances':
```

Comment:
You also need to add `'utterances'` to the list on [line 102 of `nikola.py`](https://github.com/getnikola/nikola/blob/eea7dbdb0a25a7d154f35068d77c5c2e2b975c13/nikola/nikola.py#L102).